resources/ext/ezbake.conf

```diff
@@ -4,9 +4,9 @@ ezbake: {
     debian: { dependencies: ["pe-puppet (>= 4.0.0-1puppetlabs1)"] }
```

Comment:
@kbarber @haus the pe-puppet changes above are the ones I'm not sure about, seems like we should just leave it for now?